apps/aecore/test/aec_peers_pool_tests.erl

```diff
@@ -1514,6 +1520,7 @@ validate_counters() ->
 
     % Delete all the peers.
     Pool4 = lists:foldl(fun({I, _}, P) ->
+        erlang:display(P, I),
```

Comment:
TIL: `Erlang:display/1` - thanks!

Reply:
Dhoh! I should remove that. It adds nothing to the pass/fail nature of the tests themselves.